eos-tech-support/eos-enable-debian-apt

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+echo "This script adds the Debian repository to your apt config."
+echo "The resulting configuration prefers the Endless repository where"
+echo "possible, even if a package in Endless is older than in Debian."
+echo "However, but packages not found in the Endless repo will be automatically"
+echo "downloaded from Debian".
+echo
+echo "Although this may work fine in some cases, you're likely to find issues"
+echo "where a package in Debian is somehow incompatible with Endless packages"
+echo "and this will most frequently appear as dependency resolution issues."
+echo "As such, this is an unsupported, developer-only tool."
+echo
+echo "Additionally, apt will only work after you have taken the usual steps of"
+echo "making it available (e.g. eos-convert-system, eos-dev-unlock, etc)"
+echo
+
+if [[ $# < 1 ]]; then
+	echo "Usage: $0 <DEB_DISTRO>"
+	echo "DEB_DISTRO example values: stable, testing, unstable"
+	exit 1
+fi
+
+distro=$1
+
+cat > /etc/apt/preferences.d/eos-enable-debian-apt <<EOF
+Package: *
+Pin: release o=Endless
+Pin-Priority: 600
```

Comment:
I don't think you really need to raise this above the default priority of 500 since you're explicitly lowering the debian priority to 400 below. I don't think it hurts anything in normal operation, but it's a little unexpected.

Reply:
I agree, this line can be removed

Reply:
Yeah I agree, Debian to 400 should be sufficient. Did you verify that dist-upgrade doesn't do anything whacky after adding Debian eg stable? ie the pin seems to do what we expect?